matchminer/custom.py

```diff
@@ -1427,8 +1431,7 @@ def run_ctims_matchengine_job(trial_internal_ids):
     ) as me:
         me.get_matches_for_all_trials()
         me.update_all_matches()
+        if (len(me.failed_protocol_nos.keys()) > 0):
+            return me.failed_protocol_nos
 
-    resp = Response(response=json.dumps({"success": True}),
-                    status=200,
-                    mimetype="application/json")
-    return resp
+    return failed_protocol_nos
```

Comment:
This will return failed_protocol_nos twice?

Reply:
I changed the code.